DigitalLearningSolutions.Web/ViewModels/LearningMenu/DiagnosticContentViewModel.cs

```diff
@@ -12,7 +12,8 @@ public class DiagnosticContentViewModel
         public string CourseTitle { get; }
         public string SectionName { get; }
         public string ContentSource { get; }
-        public List<int> Tutorials { get; }
+        private const string type = "diag";
+        private List<int> Tutorials { get; }
         private int CentreId { get; }
         private int CandidateId { get; }
         private int ProgressId { get; }
```

Comment:
Actually you don't need any of these private variables do you? You can just pass them to the methods which need them in the constructor. No need to store something in the class if you're just going to use it in the constructor 🙂

Reply:
In that case should I remove those two methods in the class and just call the helper methods instead?

Reply:
You could do 🙂